### ISSUE TITLE
Build failure with GHC 7.8

### ISSUE THREAD

**Issue by snoyberg** [RaphaelJ/repa-devil#7]
Subset of the output:

```
Unpacking to repa-devil-0.3.2.3/
Resolving dependencies...
Configuring repa-devil-0.3.2.3...
Building repa-devil-0.3.2.3...
Preprocessing library repa-devil-0.3.2.3...
[1 of 1] Compiling Data.Array.Repa.IO.DevIL ( dist/build/Data/Array/Repa/IO/DevIL.hs, dist/build/Data/Array/Repa/IO/DevIL.o )

Data/Array/Repa/IO/DevIL.hsc:92:40:
    Not in scope: type constructor or class ‘Applicative’

Data/Array/Repa/IO/DevIL.hsc:180:24: Not in scope: ‘<$>’

Data/Array/Repa/IO/DevIL.hsc:212:31: Not in scope: ‘<$>’

Data/Array/Repa/IO/DevIL.hsc:217:17: Not in scope: ‘<$>’

Data/Array/Repa/IO/DevIL.hsc:220:18: Not in scope: ‘<$>’

Data/Array/Repa/IO/DevIL.hsc:223:17: Not in scope: ‘<$>’

Data/Array/Repa/IO/DevIL.hsc:226:18: Not in scope: ‘<$>’

Data/Array/Repa/IO/DevIL.hsc:229:18: Not in scope: ‘<$>’

Data/Array/Repa/IO/DevIL.hsc:232:18: Not in scope: ‘<$>’

Data/Array/Repa/IO/DevIL.hsc:247:27: Not in scope: ‘<$>’

Data/Array/Repa/IO/DevIL.hsc:261:15: Not in scope: ‘<$>’

Data/Array/Repa/IO/DevIL.hsc:266:15: Not in scope: ‘<$>’

Data/Array/Repa/IO/DevIL.hsc:271:15: Not in scope: ‘<$>’

Data/Array/Repa/IO/DevIL.hsc:276:15: Not in scope: ‘<$>’

Data/Array/Repa/IO/DevIL.hsc:281:15: Not in scope: ‘<$>’
```

**Comment by RaphaelJ** [comment]
Should be fixed in v0.3.2.4 (d9314b5).

**Comment by snoyberg** [comment]
Looks like the indentation caused trouble, I'm now getting:

```
[1 of 1] Compiling Data.Array.Repa.IO.DevIL ( dist/build/Data/Array/Repa/IO/DevIL.hs, dist/build/Data/Array/Repa/IO/DevIL.o )

Data/Array/Repa/IO/DevIL.hsc:51:1: parse error on input ‘import’
```

**Comment by RaphaelJ** [comment]
Yeah, a pretty stupid error.
Fixed in v0.3.2.5 (addb516)

**Comment by RaphaelJ** [comment]
Yeah, a pretty stupid error.
Fixed in v0.3.2.5 (83a25fa).

**Comment by snoyberg** [comment]
No, still broken:

```
Unpacking to repa-devil-0.3.2.5/
Resolving dependencies...
Configuring repa-devil-0.3.2.5...
Building repa-devil-0.3.2.5...
Preprocessing library repa-devil-0.3.2.5...
[1 of 1] Compiling Data.Array.Repa.IO.DevIL ( dist/build/Data/Array/Repa/IO/DevIL.hs, dist/build/Data/Array/Repa/IO/DevIL.o )

Data/Array/Repa/IO/DevIL.hsc:96:40:
    Not in scope: type constructor or class ‘Applicative’
```

**Comment by snoyberg** [comment]
I've pegged the repa-devil version in Stackage at 0.3.2.2, the last known working version.

**Comment by RaphaelJ** [comment]
I fixed that in 6860a6b and its successfully compiled with GHC 7.8.
